src/components/MessageList/utils/constructTypingString.js

```diff
@@ -0,0 +1,38 @@
+export const constructTypingString = ({ client, t, typingUsers }) => {
```

Comment:
How about we make it a hook? Easy to reuse for customers who don't use our TypingIndicator?

Reply:
Done